services/ConfigService.go

```diff
@@ -82,6 +84,11 @@ func (c ConfigService) getConfigInteractive() configAnswers {
 			Prompt:   &survey.Input{Message: "What key is your project using (e.g TFLOW)?"},
 			Validate: survey.Required,
 		},
+		{
+			Name:     "branchSeparator",
+			Prompt:   &survey.Input{Message: "How do you want to separate the Jira key from the Jira issue title in your branch name? Most people use a slash ('/') here."},
```

Comment:
I feel like this would steer most people without a preference in the direction of `/` which you mentioned is problematic with git in a rather nontransparent way. Maybe we should help these people as well and suggest to use `|` as a default (or maybe something else if you have a suggestion?).

What do you think?

Reply:
Yeah I've thought about that, my choice for `/` as the default was that a lot of people seem to be using it and I wasn't sure if it would make sense to nudge people into using something else without explaining why. Explaining why would be rather hard/verbose during the survey I think. Do you have a suggestion?

further: I was kinda hoping that `survey` offered a option for setting defaults (that you can accept by pressing enter or something). It seems such a thing is not implemented.

Reply:
>Yeah I've thought about that, my choice for / as the default was that a lot of people seem to be using it and I wasn't sure if it would make sense to nudge people into using something else without explaining why. Explaining why would be rather hard/verbose during the survey I think. Do you have a suggestion?

If you mean people coming from gitflow etc I think we shouldn't have their default as our default because most of them will probably not be aware about this problem with git. At least for me, the only reason why I was using `/` was because of gitflow way back so I can imagine many more people are doing this out of habit. In that sense it would be nice to spread some knowledge. If people really want to use `/` they are completely free to do so.

>further: I was kinda hoping that survey offered a option for setting defaults (that you can accept by pressing enter or something). It seems such a thing is not implemented.

I think this can be implemented with relative ease. If you include something like `(default: |)` in the question and remove the `validate.surveyRequired` then before you set it on line `46` you can just check whether  `answers.Branchseperator` is nil and if it is use the default value.